bin/protogen

```diff
@@ -34,7 +34,8 @@ JOIN = os.path.join
 TOP_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 
 
-def main(args=sys.argv[1:]):
+def main(args=None):
+    args = sys.argv[1:] if args is None else args
```

Comment:
Avoid using conditional expressions as replacements for simplistic flow control statements.  This is better:

```
if args is None:
    args = sys.argv[1:]
```

This is much clearer and technically correct (doesn't re-assign args when it not None).  It is also fewer tokens to read total and fewer tokens-per-line.